src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_x509chain.c

```diff
@@ -192,7 +192,7 @@ static void MergeStatusCodes(CFTypeRef key, CFTypeRef value, void* context)
     }
     else
     {
-#ifdef DEBUGGING_UNKNOWN_VALUE
+#ifdef DEBUGGING_UNKNOWN_VALUE || DEBUG
```

Comment:
I got a syntax error when I built this... since only the left token is a "#if defined"

Reply:
Ah. Closing in favor of yours then.